pkg/util/lifted/lifted_record_list.go

```diff
@@ -0,0 +1,64 @@
+package lifted
```

Comment:
```suggestion
// Package lifted contains the files lifted from other projects.
package lifted
```

And rename the file name to `docs.go`